api/zeebe/generation-strategy.js

```diff
@@ -0,0 +1,28 @@
+const replace = require("replace-in-file");
+
+const outputDir = "docs/apis-tools/zeebe-api-rest/specifications";
+function preGenerateDocs() {
+  // noop
```

Comment:
The `preGenerateDocs()` hook is obviously not yet needed in this PR, because I've defined it as a noop, but other APIs will need this hook. 

I could make the logic in `generate-api-docs` smart enough to not explode if there's no `preGenerateDocs()` function defined by the strategy, but I like the deliberate and explicit implementation of a specific interface here.